test/unit/modules/signatures.js

```diff
@@ -14,66 +14,226 @@
+			library.rewiredModules.signatures.__set__('modules', {
+				accounts: library.modules.accounts,
+				transactions: library.modules.transactions,
+				transport: library.modules.transport,
+			});
+			expect(library.modules.signatures.isLoaded()).to.be.true;
+			done();
+		});
 
-		it('should return true if modules does not exist');
+		it('should return true if modules does not exist', done => {
+			const revert = library.rewiredModules.signatures.__set__(
+				'modules',
+				undefined
+			);
+			expect(library.modules.signatures.isLoaded()).to.be.false;
+			revert();
+			done();
+		});
 	});
 
 	describe('onBind', () => {
+		let privateModules;
+		let signatureLogicSpy;
+
+		before(done => {
+			privateModules = library.rewiredModules.signatures.__get__('modules');
+			signatureLogicSpy = sinonSandbox.spy(
+				library.rewiredModules.signatures.__get__('__private').assetTypes[
+					transactionTypes.SIGNATURE
+				],
+				'bind'
+			);
+			library.modules.signatures.onBind(library.modules);
+			done();
+		});
+
 		describe('modules', () => {
-			it('should assign accounts');
+			it('should assign accounts', done => {
+				expect(privateModules).to.have.property(
+					'accounts',
+					library.modules.accounts
+				);
+				done();
+			});
 
-			it('should assign transactions');
+			it('should assign transactions', done => {
+				expect(privateModules).to.have.property(
+					'transactions',
+					library.modules.transactions
+				);
+				done();
+			});
 
-			it('should assign transport');
+			it('should assign transport', done => {
+				expect(privateModules).to.have.property(
+					'transport',
+					library.modules.transport
+				);
+				done();
+			});
 		});
 
 		describe('assetTypes', () => {
-			it('should call bind on signature logic with scope.accounts');
+			it('should call bind on signature logic with scope.accounts', done => {
+				expect(signatureLogicSpy).to.be.calledWith(library.modules.accounts);
+				done();
+			});
 		});
 	});
 
 	describe('shared', () => {
-		describe('postSignatures', () => {
-			describe('when modules not are loaded', () => {
-				it('should call callback with ApiError');
-
-				it(
-					'should call callback with ApiError containing message = "Blockchain is loading"'
+		describe('postSignature', () => {
+			const req = {
+				body: {
+					signature: 'aSignature',
+					transactionId: 'aTransactionId',
+					publicKey: 'aPublicKey',
+				},
+			};
+
+			let postSignatureSpy;
+
+			before(done => {
+				postSignatureSpy = sinonSandbox.spy(
+					library.modules.transport.shared,
+					'postSignature'
+				);
+				library.modules.signatures.shared.postSignature(
+					req.body,
+					(error, response) => {
+						done();
+					}
 				);
+			});
 
-				it('should call callback with ApiError containing code = 500');
+			it('should call modules.transport.shared.postSignature with req.body', done => {
+				expect(postSignatureSpy).to.be.calledWith({ signature: req.body });
+				sinonSandbox.restore();
```

Comment:
please restore the sandbox in after hook. And restoring `sinonSandbox` here is not ok since you didn't use `beforeEach` but `before`

Reply:
Done

Reply:
Done